base/src/com/google/idea/blaze/base/actions/FileGitHubIssueAction.java

```diff
@@ -62,7 +62,10 @@ public final class FileGitHubIssueAction extends BlazeProjectAction {
   private static final NotNullLazyValue<Boolean> isCorpMachine =
       NotNullLazyValue.createValue(
           () -> {
-            int retVal = ExternalTask.builder().args("glogin", "-version").build().run();
+            int retVal = ExternalTask.builder().args("which", "glogin").build().run();
```

Comment:
I think this will break on Windows, or at least throw an error - which we're trying to avoid. Maybe it's expected to return false on Windows anyway, but we should check.

Reply:
Never mind. It turns out that Bazel has some support for Windows. I have changed my code accordingly.